platform/ui/src/components/SidePanel/SidePanel.tsx

```diff
@@ -168,171 +182,171 @@ const SidePanel = ({ side, className, activeTabIndex: activeTabIndexProp, tabs,
   const getCloseStateComponent = () => {
     const _childComponents = Array.isArray(tabs) ? tabs : [tabs];
     return (
-      <>
-        <div
-          className={classnames(
-            'bg-secondary-dark flex h-[28px] w-full cursor-pointer items-center rounded-md',
-            side === 'left' ? 'justify-end pr-2' : 'justify-start pl-2'
-          )}
-          onClick={() => {
-            updatePanelOpen(prev => !prev);
-          }}
-          data-cy={`side-panel-header-${side}`}
-        >
-          <Icon
-            name={'navigation-panel-right-reveal'}
-            className={classnames('text-primary-active', side === 'left' && 'rotate-180 transform')}
-          />
-        </div>
-        <div className={classnames('mt-3 flex flex-col space-y-3')}>
-          {_childComponents.map((childComponent, index) => (
-            <Tooltip
-              position={side === 'left' ? 'right' : 'left'}
-              key={index}
-              content={`${childComponent.label}`}
+        <>
```

Comment:
I see a lot of places like this where the indentation changed and I am not sure why. Could you please fix this?